src/ArcGISRuntime.Samples.Shared/Resources/FeaturedSamples.xml

```diff
@@ -4,6 +4,7 @@
     <Category name="Data">
       <Sample>AddFeaturesWithContingentValues</Sample>
       <Sample>ToggleBetweenFeatureRequestModes</Sample>
+      <Sample>CreateMobileGeodatabase</Sample>
```

Comment:
Very minor nitpick, this might be better placed above `ToggleBetweenFeatureRequestModes` just to maintain alphabetical order.